WordPress/Sniffs/VIP/RestrictedFunctionsSniff.php

```diff
@@ -122,34 +113,16 @@ public function getGroups() {
 
 			'attachment_url_to_postid' => array(
 				'type'      => 'error',
-				'message'   => '%s is prohibited, please use wpcom_vip_attachment_url_to_postid() instead.',
+				'message'   => '%s() is prohibited, please use wpcom_vip_attachment_url_to_postid() instead.',
 				'functions' => array(
 					'attachment_url_to_postid',
 				),
 			),
 
-			// @link https://vip.wordpress.com/documentation/vip/code-review-what-we-look-for/#remote-calls
-			'wp_remote_get' => array(
```

Comment:
This section looks to be removed without it being brought back anywhere... Looks like a mistake ?